src/util/random-string.ts

```diff
@@ -1,4 +1,15 @@
 export const createRandomString = () => {
+    if (
+        !('crypto' in window) ||
+        typeof window.crypto?.randomUUID !== 'function'
+    ) {
+        return legacyCreateRandomString();
+    }
+
+    return crypto.randomUUID().replace(/^\d/, 'a'); // ids must start with letters
```

Comment:
Any particular reason why you've chosen to regex match for a digit and then replace it if found, rather than just prepending all UUIDs output with a letter? Seems like that would be both faster and simpler? 😊

Reply:
I like to live on the edge 😅, no good reason. It felt off to me to change the length/format of the id but there's no real reason to worry about that

Reply:
I personally prefer to avoid using a regular expression, if I can easily do the same thing with common string operations.

Reply:
I'd even argue that it's better to break the UUID spec, since it's no longer a UUID when you change one of the characters to a fixed value. We might even want to prefix it with something like `a_` to make it even more obvious that it's not a UUID.

Reply:
That is a good point. However the consumer of `createRandomString` has no idea about `UUID` and it doesn't care if the random string is UUID or something else 😆.  Of course it becomes clear/readable and clean if we do it according to your suggestion. We strive for the best